### ISSUE TITLE
Git link copy button not getting URL onto clipboard, Linux Mint+Chrome

### ISSUE THREAD

**Issue by JasonThomasData** [openaustralia/morph#1152]
I think the desired UX is for this button:

to get the Git link onto the clipboard. Obviously not a major issue, but this has not worked on my PC today.

I use Linux Mint Sarah, and Google Chrome. Interestingly, this button does work for me with Firefox, so maybe won't affect many.

**Comment by chris48s** [comment]
Yeah - I get the same behaviour on Ubuntu - works in Firefox, not in Chromium.

Having a quick dig about, it looks like there are some bug reports from a couple of years ago about this:
https://github.com/zeroclipboard/zeroclipboard-rails/issues/32
https://github.com/zeroclipboard/zeroclipboard-rails/issues/33
but the links provided are about how to get Flash working in Chrome/Chromium on Ubuntu.

Interestingly GitHub are also using ZeroClipboard and their copy link does work in Chromium-derived browsers so it might be worth upgrading the library as a first step to see if that fixes it. It looks like ZeroClipboard is only used in one page on morph, so bumping `zeroclipboard-rails` up to the latest version should be pretty low-risk.

**Comment by benrfairless** [comment]
Thanks for raising this, and sorry it has sat without a response for so long.

We are tidying up the morph.io issue tracker. This issue dates from 2017 and, on review, looks to have gone stale (the idea has been overtaken, superseded, or not picked up in the years since), so we are closing it as part of a backlog review to keep the tracker focused on work we can realistically act on. Closing it is not a judgement that the underlying idea was wrong.

If this is still relevant to you, please reopen it or open a fresh issue with up to date detail and we will gladly take another look.

This triage was carried out with AI assistance (Claude Opus 4.8 via Claude Code) and reviewed by the OpenAustralia Foundation team.
